assets/src/edit-story/components/library/panes/media/common/styles.js

```diff
@@ -36,16 +37,32 @@ export const PaneHeader = styled.div`
 `;
 
 export const MediaGalleryContainer = styled.div`
+  display: grid;
   grid-area: infinitescroll;
   overflow: auto;
-  padding: 0 24px;
+  grid-template-columns: 1fr;
+  padding: 0 1.2em 0 1.2em;
```

Comment:
As discussed offline, it's better to use px's here to facilitate the calculation of the inner container with fixed width.